LottieSample/src/main/java/com/airbnb/lottie/samples/AnimationFragment.java

```diff
@@ -94,10 +94,10 @@ static AnimationFragment newInstance() {
   @BindView(R.id.play_button) ImageButton playButton;
   @BindView(R.id.loop) ImageButton loopButton;
   @BindView(R.id.animation_name) TextView animationNameView;
-  @BindView(R.id.qr_code_textview) TextView qrCodeTextView;
-  @BindView(R.id.sample_animations_textview) TextView sampleAnimationsTextView;
-  @BindView(R.id.load_animation_textview) TextView loadAnimationTextView;
-  @BindView(R.id.load_from_json_textview) TextView loadFromJsonTextView;
+  @BindView(R.id.qr_code) TextView qrCode;
```

Comment:
You can leave the variable names as they were and just change the R values as you have done.